owncloudApp/src/androidTest/java/com/owncloud/android/settings/security/OCSettingsPasscodeTest.kt

```diff
@@ -192,6 +190,7 @@ class OCSettingsPasscodeTest {
         typePasscode(wrongPassCode)
 
         onView(withText(R.string.pass_code_enter_pass_code)).check(matches(isDisplayed()))
+        onView(withText(R.string.pass_code_wrong)).check(matches(isDisplayed()))
```

Comment:
Since we are showing this text in the layout, I would try to use `withId` instead of `withText`. That way, we will check that the text is shown in that specific TextView. Otherwise, we will check that that text is shown somewhere and is not as accurate.

Let's give it a try 🍻

Reply:
Ok, I adapted the rest of tests also